target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -168,6 +168,13 @@ glinet,gl-mifi)
 glinet,gl-x750)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
 	;;
+meraki,mr16)
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth0"
+	ucidef_set_led_wlan "wlan1" "WLAN1" "$boardname:green:wifi1" "phy0assoc"
+	ucidef_set_led_wlan "wlan2" "WLAN2" "$boardname:green:wifi2" "phy0assoc"
+	ucidef_set_led_wlan "wlan3" "WLAN3" "$boardname:green:wifi3" "phy0assoc"
+	ucidef_set_led_wlan "wlan4" "WLAN4" "$boardname:green:wifi4" "phy0tpt"
+	;;
```

Comment:
We setup these LEDs via the board.d, vs the device tree, so users can re-map them to whatever purpose they want without having to "create" LED defines in Luci/uci.